src/postgres/commands/deletePostgresDatabase.ts

```diff
@@ -17,4 +18,7 @@ export async function deletePostgresDatabase(context: IActionContext, node?: Pos
     if (result === DialogResponses.deleteResponse) {
         await node.deleteTreeItem(context);
     }
+    const deleteMessage = localize('deletePostgresDatabaseMsg', 'Successfully deleted database "{0}".', node.databaseName);
```

Comment:
Would it be better to put this logging in `deleteTreeItemImpl`? @EricJizbaMSFT may have thoughts on this

Reply:
I actually prefer it here for two reasons:
1. Since the tree item files are shared in so many places, I'd like to keep as much code out of there as possible
1. It's better to have the confirmation dialog outside of `deleteTreeItemImpl` so that we don't get the "Deleting..." thing in the tree while we wait for the user to confirm deletion. And it feels consistent to put the completed notification here if that's where the confirmation dialog is